Nupkg.props

```diff
@@ -0,0 +1,47 @@
+<?xml version="1.0" encoding="utf-8"?>
+<Project xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+  <PropertyGroup>
+    <!-- Including this file will generate both the *.nupkg and *.symbols.nupkg -->
+    <GeneratePackageOnBuild>True</GeneratePackageOnBuild>
+    <IncludeSymbols>True</IncludeSymbols>
+  </PropertyGroup>
+
+  <PropertyGroup>
+    <!-- These are the common properties used when generating the nupkg -->
+    <!-- https://docs.microsoft.com/en-us/nuget/schema/msbuild-targets -->
+    <Company>Microsoft</Company>
+    <Copyright>© Microsoft Corporation. All rights reserved.</Copyright>
+    <!-- <PackageVersion>Defined in GlobalStaticVersion.props</PackageVersion> -->
+    <Authors>Microsoft</Authors>
+    <Owners>Microsoft,AppInsightsSdk</Owners>
+    <PackageRequireLicenseAcceptance>true</PackageRequireLicenseAcceptance>
+    <PackageLicenseUrl>https://go.microsoft.com/fwlink/?LinkID=510709</PackageLicenseUrl>
+    <PackageProjectUrl>https://go.microsoft.com/fwlink/?LinkId=392727</PackageProjectUrl>
+    <PackageIconUrl>http://appanacdn.blob.core.windows.net/cdn/icons/aic.png</PackageIconUrl>
+    <RepositoryUrl>https://github.com/Microsoft/ApplicationInsights-dotnet</RepositoryUrl>
+    <RepositoryType>Git</RepositoryType>
+    <PackageType>Dependency</PackageType>
+    <ContentTargetFolders>content</ContentTargetFolders>
+    <PackageReleaseNotes>For the release notes please follow http://go.microsoft.com/fwlink/?LinkId=535037</PackageReleaseNotes>
+    <!-- <PackageOutputPath>Defined in Directory.Build.props</PackageOutputPath> -->
+  </PropertyGroup>
+
+  <PropertyGroup>
+    <!-- These Properties are unique to the project and must be set in the csproj -->
+    <PackageId>UNDEFINED</PackageId>
+    <Title>UNDEFINED</Title>
+    <Description>UNDEFINED</Description>
+    <PackageTags>UNDEFINED</PackageTags>
+  </PropertyGroup>
+
+  <PropertyGroup>
+    <!-- Disable auto generation of package attributes. This resolves the 'Duplicate attribute' error. 
+         Common cause is a hidden AssemblyInfo.cs file in a project -->
+    <GenerateAssemblyVersionAttribute>false</GenerateAssemblyVersionAttribute>
```

Comment:
we probably will need to get rid of those at some point... It is set to not clean up the custom code we created with props files before it was supported =)